### ISSUE TITLE
[Information Request] Current state of this addon

### ISSUE THREAD

**Issue by Mannshoch** [owncloud/mozilla_sync#149]
Hello Developers of this nice tool

Please Update your README.md
Tell us Please whats the current state of this addon. How far is the support of the current Protocol. Whats your problems, roadmap and difficulties. Could we somehow support you?

Best regards,
Mannshoch

**Comment by RealRancor** [comment]
@Mannshoch I don't think that anyone is currently developing this app. It needs a completely re-write of the app to support the new mozilla sync version like discussed at various places at the tracker here (see https://github.com/owncloud/mozilla_sync/issues/33).

**Comment by Mannshoch** [comment]
Even it is as you say an Official Information would also be a nice touch.
